bazel/switched_rules_extension.bzl

```diff
@@ -0,0 +1,149 @@
+load("@com_google_googleapis//:repository_rules.bzl", "switched_rules")
+# -- load statements -- #
+
+def _switched_rules_extension_impl(ctx):
+    switched_rules(
```

Comment:
Why not use `switched_rules_by_language` like we used to do? This seems brittle.

Reply:
It's done now